doc/script_commands.txt

```diff
@@ -4083,18 +4083,20 @@ Note: rid2name may not produce correct character names since rid means
 
 ---------------------------------------
 
+*message <account ID>,"<message>";
 *message "<character name>","<message>";
 
 That command will send a message to the chat window of the character 
-specified by name. The text will also appear above the head of that 
-character. It will not be seen by anyone else.
+specified by account ID or name. The text will also appear above the head
+of that character. It will not be seen by anyone else.
 
 ---------------------------------------
 
-*dispbottom "<message>";
+*dispbottom "<message>",{<color>,{<account ID>}};
```

Comment:
To be honest, I'd prefer it if we left out the 'account ID' optional argument, for various reasons:

- There is no way to specify the account ID without specifying the color, making it half useless.
- Either all commands or no commands should be able to act on a separate character (and I'd prefer no commands *).

*: The reason why I said I prefer that no commands had this option, is that:
- You can already attachrid (and there's very little reason for a script to act on a non-attached character in any case)
- If we were to do it, we'd need an unified way for all commands, to report whether the given character was online or not .Using attachrid instead, you can check whether the attach command was successful.